bluemira/equilibria/optimisation/problem/_nested_position.py

```diff
@@ -87,15 +84,13 @@ def __init__(
         self,
         sub_opt: CoilsetOptimisationProblem,
         eq: Equilibrium,
-        targets: MagneticConstraintSet,
```

Comment:
Why is this removed?

Reply:
They aren't used.

This COP is doing pos. opt. only. It does not use a magnetic constraint set.
(the sub_opt may have one/use one but that would be handled entirely within the sub_opt)